app/src/layouts/appLayout/appHeader/appHeader.jsx

```diff
@@ -57,7 +57,12 @@ export class AppHeader extends Component {
       <header className={cx('header')}>
         <MobileHeader opened={sideMenuOpened} toggleSideMenu={toggleSideMenu} />
         <div className={cx('projects-block')}>
-          <ProjectSelector projects={projects} activeProject={activeProject} mobileOnly />
+          <ProjectSelector
+            projects={projects}
+            assignedProjects={assignedProjects}
```

Comment:
May be calculate and sort project keys on the `ProjectSelector` level?

Reply:
I thought about this.
It will be better solution.